src/main/java/fr/pilato/elasticsearch/tools/pipeline/PipelineElasticsearchUpdater.java

```diff
@@ -0,0 +1,128 @@
+package fr.pilato.elasticsearch.tools.pipeline;
+
+import java.io.IOException;
+
+import org.elasticsearch.client.Request;
+import org.elasticsearch.client.Response;
+import org.elasticsearch.client.ResponseException;
+import org.elasticsearch.client.RestClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Handles ingest pipeline creation.
+ * 
+ * @author hjk181
+ *
+ */
+public class PipelineElasticsearchUpdater {
+	
+	private static final Logger logger = LoggerFactory.getLogger(PipelineElasticsearchUpdater.class);
+
+	/**
+	 * Create a pipeline in Elasticsearch.
+	 * @param client Elasticsearch client
+	 * @param root dir within the classpath
+	 * @param pipeline the id of the pipeline
+     * @param force set it to true if you want to force cleaning pipeline before adding it
+     * @throws Exception if something goes wrong
+	 */
+	public static void createPipeline(RestClient client, String root, String pipeline, boolean force) throws Exception {
+		String json = PipelineSettingsReader.readPipeline(root, pipeline);
+		createPipelineWithJson(client, pipeline, json, force);
+	}
+
+	/**
+	 * Create a pipeline in Elasticsearch. Read content from default classpath dir.
+	 * @param client Elasticsearch client
+	 * @param pipeline the id of the pipeline
+     * @param force set it to true if you want to force cleaning pipeline before adding it
+     * @throws Exception if something goes wrong
+	 */
+	public static void createPipeline(RestClient client, String pipeline, boolean force) throws Exception {
+		String json = PipelineSettingsReader.readPipeline(pipeline);
+		createPipelineWithJson(client, pipeline, json, force);
+	}
+
+	/**
+	 * Create a new pipeline in Elasticsearch
+	 * @param client Elasticsearch client
+	 * @param pipeline the id of the pipeline
+	 * @param json JSon content for the pipeline
+	 * @param force set it to true if you want to force cleaning pipeline before adding it
+     * @throws Exception if something goes wrong
+	 */
+	public static void createPipelineWithJson(RestClient client, String pipeline, String json, boolean force) throws Exception {
+		if (isPipelineExist(client, pipeline)) {
```

Comment:
I think we could simplify all this and that it's not needed to have a `removePipeline()` function unless we need it for tests.
Because elasticsearch can totally overwrite an existing pipeline.